VirtualList/tests/VirtualList-translate-specs.js

```diff
@@ -0,0 +1,811 @@
+				const actual = resultScrollTop;
+
+				expect(actual).toBe(expected);
+			});
+
+			const {rerender} = render(
+				<VirtualList
+					cbScrollTo={getScrollTo}
+					clientSize={clientSize}
+					dataSize={dataSize}
+					itemRenderer={renderItem}
+					itemSize={itemSize}
+					onScrollStop={onScrollStop}
+					scrollMode="translate"
+				/>
+			);
+
+			rerender(
+				<VirtualList
+					cbScrollTo={getScrollTo}
+					clientSize={clientSize}
+					dataSize={dataSize}
+					itemRenderer={renderItem}
+					itemSize={newItemSize}
+					onScrollStop={onScrollStop}
+					scrollMode="translate"
+				/>
+			);
+
+			act(() => myScrollTo({align: 'bottom', animate: false}));
+		});
+
+		test('should scroll to the given align with scrollTo after changing spacing', (done) => {
+			const newSpacing = 30;
+
+			const onScrollStop = handlerOnScrollStop(done, () => {
+				const expected = dataSize * (itemSize + newSpacing) - clientSize.clientHeight - newSpacing + 30;
+				const actual = resultScrollTop;
+
+				expect(actual).toBe(expected);
+			});
+
+			const {rerender} = render(
+				<VirtualList
+					cbScrollTo={getScrollTo}
+					clientSize={clientSize}
+					dataSize={dataSize}
+					itemRenderer={renderItem}
+					itemSize={itemSize}
+					onScrollStop={onScrollStop}
+					scrollMode="translate"
+				/>
+			);
+
+			rerender(
+				<VirtualList
+					cbScrollTo={getScrollTo}
+					clientSize={clientSize}
+					dataSize={dataSize}
+					itemRenderer={renderItem}
+					itemSize={itemSize}
+					onScrollStop={onScrollStop}
+					spacing={newSpacing}
+					scrollMode="translate"
+				/>
+			);
+
+			act(() => myScrollTo({align: 'bottom', animate: false}));
+		});
+
+		test('should scroll to the given align with scrollTo when itemSizes is given', (done) => {
+			const consoleSpy = jest.spyOn(console, 'error').mockImplementation(() => {});
+			const itemSizes = [100, 200, 300, 400, 100, 200, 300, 400, 100, 200];
+			const itemSizeSum = itemSizes.reduce((sum, value) => sum + value); // 2300
+			const onScrollStop = handlerOnScrollStop(done, () => {
+				const expected = itemSizeSum - clientSize.clientHeight + 30;
+				const actual = resultScrollTop;
+
+				expect(actual).toBe(expected);
+			});
+
+			const {rerender} = render(
+				<VirtualList
+					cbScrollTo={getScrollTo}
+					clientSize={clientSize}
+					dataSize={10}
+					itemRenderer={renderItem}
+					itemSize={itemSize}
+					onScrollStop={onScrollStop}
+					scrollMode="translate"
+				/>
+			);
+
+			rerender(
+				<VirtualList
+					cbScrollTo={getScrollTo}
+					clientSize={clientSize}
+					dataSize={dataSize}
+					itemRenderer={renderItem}
+					itemSize={itemSize}
+					itemSizes={itemSizes}
+					onScrollStop={onScrollStop}
+					scrollMode="translate"
+				/>
+			);
+
+			act(() => myScrollTo({align: 'bottom', animate: false}));
+
+			const expectedErrorMsg = 'Warning: VirtualList with `minSize` in `itemSize` prop does not support `cbScrollTo` prop';
```

Comment:
Should we check this warning?

Reply:
We don't need to check bu I wanted to leave an explanation as to why the warning occurs. I've chosen to verify messages rather than comments.

Reply:
It would be better if we just put a comment about the warning.

Reply:
As we discussed, I think we don't have much information about this warning. Let's skip the test now and take a look at this in detail in another ticket.

Reply:
I set "skip" and made WRP-4365